src/Language/PureScript/Docs/Types.hs

```diff
@@ -184,6 +185,15 @@ data DeclarationInfo
 
 instance NFData DeclarationInfo
 
+-- |
+-- Wraps enough information to properly render the kind signature
+-- of a data/newtype/type/class declaration.
+data KindInfo
+  = KindInfo P.KindSignatureFor Type'
```

Comment:
Can we make this into a record type with named parameters please? I know that this is already a common pattern in the compiler, but having to remember what the nth parameter of each data constructor means kind of sucks.

Reply:
Sure.

If you'd like, I could slowly update this codebase to use records where possible, bit by bit.

Reply:
I've opened #4115 to track this. I'll mark this as resolved.